### PR TITLE
feat: update translations

### PR DESCRIPTION
**Pull request by scottdover**
**Summary**
This updates our translations for recent work

**Testing**
👀

### CODE REVIEW

File: l10n/bundle.l10n.de.json [hunk 1]
```diff
@@ -4,16 +4,22 @@
   "Cancelling job...": "Abbruch des Jobs...",
   "Cannot call self on ComputeSession with no id": "Kann self nicht für ComputeSessions ohne ID aufrufen",
   "Cannot call self on object with no id": "Kann self für ein Object ohne ID nicht aufrufen",
+  "Cannot connect to SAS Studio service": "Cannot connect to SAS Studio service",
```
Comment (scottdover) [inline l10n/bundle.l10n.de.json:7]: @Criptic  Do you mind providing translations for the new terms, here?

File: l10n/bundle.l10n.zh-cn.json [hunk 2]
```diff
@@ -7,12 +7,15 @@
   "Cannot call self on object with no id": "无法在没有id的对象上调用self",
   "Cannot connect to SAS Studio service": "无法连接到 SAS Studio 服务",
   "Cannot find file: {file}": "无法找到文件: {file}",
+  "Choose where to save your files.": "Choose where to save your files.",
```
Comment (scottdover) [inline l10n/bundle.l10n.zh-cn.json:10]: @scnwwu  Do you mind providing translations for new terms here?

File: package.nls.ko.json [hunk 3]
```diff
@@ -23,6 +23,9 @@
   "commands.SAS.runSelected": "선택한 코드 또는 모든 SAS 코드 실행",
   "commands.SAS.switchProfile": "현재 연결 프로필 전환",
   "commands.SAS.updateProfile": "연결 프로필 업데이트",
+  "commands.SAS.upload": "Upload",
```
Comment (scottdover) [inline package.nls.ko.json:26]: @meilanji Do you mind providing translations for new terms here?

File: package.nls.zh-cn.json [hunk 4]
```diff
@@ -23,6 +23,9 @@
   "commands.SAS.runSelected": "运行选定或所有 SAS 代码",
   "commands.SAS.switchProfile": "切换当前连接配置文件",
   "commands.SAS.updateProfile": "更新连接配置文件",
+  "commands.SAS.upload": "Upload",
```
Comment (scottdover) [inline package.nls.zh-cn.json:26]: @scnwwu  Do you mind providing translations for new terms here?

**Review by scnwwu — COMMENTED** [review]
Hi @scottdover, shall we include Korean in the Locale contributors table in Contributing.md?

File: package.nls.zh-cn.json [hunk 5]
```diff
@@ -23,6 +23,9 @@
...
+  "commands.SAS.uploadFiles": "Upload File(s)",
```
Comment (scnwwu) [inline package.nls.zh-cn.json:27]: Hi @scottdover, this English string does NOT meet our i18n guidelines. Would you update this? Thanks!

[the remark this answers is not in the document]
Comment (scottdover) [reply package.nls.zh-cn.json:27]: Hey @scnwwu . I'll update the english terms to be:
 - Upload Files
 - Upload Folders

Thanks for pointing that out

File: l10n/bundle.l10n.zh-cn.json [hunk 6]
```diff
@@ -94,6 +98,7 @@
   "The item could not be added to My Favorites.": "该项目无法添加到我的收藏夹。",
   "The item could not be removed from My Favorites.": "无法从我的收藏夹中删除该项目。",
   "The notebook file does not contain any code to convert.": "笔记本文件不包含任何要转换的代码。",
+  "The notebook has been successfully converted to a flow and saved into the following folder: {folderName}. You can now open it in SAS Studio.": "The notebook has been successfully converted to a flow and saved into the following folder: {folderName}. You can now open it in SAS Studio.",
   "The notebook has been successfully converted to a flow. You can now open it in SAS Studio.": "笔记本已成功转换为流程。您现在可以在 SAS Studio 中打开它。",
```
[the remark this answers is not in the document]
Comment (scottdover) [reply l10n/bundle.l10n.zh-cn.json]: Resolved in https://github.com/sassoftware/vscode-sas-extension/pull/584/commits/ae820d5ec76fed6eae8ad5c4a772453b4f61d945

File: package.nls.zh-cn.json (hunk 5 above)
Comment (scottdover) [reply package.nls.zh-cn.json]: Resolved in https://github.com/sassoftware/vscode-sas-extension/pull/584/commits/d48bc53defda2f6a917794d2d455d6d619964ca7

**Comment by scottdover** [comment]
Instead of trying to do all the language in one go, I'm going to create a PR per language and reach out to each contributor individually.

**Review by scnwwu — APPROVED** [review]
Good to me. Thanks.

**Pull request merged**